glossary-sync.py

```diff
@@ -4,38 +4,27 @@
 from typing import Dict, Iterable, List, Optional, Set, Tuple, TypeVar, Union
 
 import click
+import progressbar
+import requests
 import ruamel.yaml.util
-from ruamel.yaml import YAML
-
-from datahub.ingestion.graph.client import (
-    DatahubClientConfig,
-    DataHubGraph,
-    get_default_graph,
-)
+from datahub.ingestion.graph.client import (DatahubClientConfig, DataHubGraph,
+                                            get_default_graph)
 from datahub.ingestion.source.metadata.business_glossary import (
```

Comment:
Import sorting looks weird. Did you use isort and black to format the code?

Reply:
yes